repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/merge/ByEntityYqlQueriesMerger.java

```diff
@@ -160,41 +162,43 @@ private MergingState doTransition(MergingState state, Statement.QueryType nextQu
         return nextState;
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"rawtypes", "unchecked"})
```

Comment:
There we can do not use SuppressWarnings
```
    private static <E extends Entity<E>> YdbRepository.Query<?> convertInsertToUpsert(YdbRepository.Query<?> query) {
        YqlStatement<?, E, ?> srcStatement = convertQueryToYqlStatement(query);
        EntitySchema<E> schema = srcStatement.getInSchema();
        TableDescriptor<E> tableDescriptor = srcStatement.getTableDescriptor();

        return new YdbRepository.Query<>(
                new UpsertYqlStatement<>(tableDescriptor, schema),
                query.getValues().get(0)
        );
    }

    private static <E extends Entity<E>> YqlStatement<?, E, ?> convertQueryToYqlStatement(YdbRepository.Query<?> query) {
        return (YqlStatement<?, E, ?>) query.getStatement();
    }
```

Reply:
Hmmm, we can use the neat trick here with generic parameter, yes :-D